M2/Macaulay2/m2/methods.m2

```diff
@@ -683,28 +683,10 @@ protect QuotientRingHook
 -- stashing or caching computed values for future reference in functions that take a mutable hash table as input
 -----------------------------------------------------------------------------
 
-CacheFunction = new Type of FunctionClosure
-CacheFunction.synonym = "a cache function"
-net CacheFunction := f -> "-*a cache function*-"
-cacheValue = key -> f -> new CacheFunction from (x -> (
-	  c := try x.cache else x.cache = new CacheTable;
-	  if c#?key then (
-	       val := c#key;
-	       if class val === CacheFunction then (
-		    remove(c,key);
-		    c#key = val x)
-	       else val
-	       )
-	  else c#key = f x))
-stashValue = key -> f -> new CacheFunction from (x -> (
-	  if x#?key then (
-	       val := x#key;
-	       if class val === CacheFunction then (
-		    remove(x,key);
-		    x#key = val x)
-	       else val
-	       )
-	  else x#key = f x))
+-- deprecated, but kept for backwards compatibility
+-- FIXME: https://github.com/Macaulay2/M2/issues/3612
```

Comment:
Is there something still outstanding with #3612?  I thought it was fixed in #3614?